packages/fast-sketch-library/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@microsoft/fast-sketch-library",
+  "description": "A Node.js utility for quickly extracting Sketch symbol libraries from a website",
+  "version": "1.0.0",
```

Comment:
Always worth another look, but I don't think that this has any side effects.

Can you add `sideEffects: false` to the package.json so that we can enable tree shaking for ourselves and others?

Reply:
sure